bin/modules/birthday/birthday_module.dart

```diff
@@ -0,0 +1,134 @@
+import 'dart:async';
+
+import 'package:cron/cron.dart';
+import 'package:hive/hive.dart';
+import 'package:nyxx/nyxx.dart';
+import 'package:nyxx_interactions/nyxx_interactions.dart';
+
+import '../bot_module.dart';
+import 'hive_constants.dart';
+
+part 'commands.dart';
+
+class BirthdayModule extends BotModule {
+  static final BirthdayModule _singleton = BirthdayModule._internal();
```

Comment:
If you use injectable : https://pub.dev/packages/injectable
It would suffice to just tag the class with @Singleton

@Singleton
class BirthdayModule